OrbitBase/include/OrbitBase/Profiling.h

```diff
@@ -38,4 +49,11 @@ inline TickType MicrosecondsToTicks(double a_Micros) {
   return static_cast<TickType>(a_Micros * 1000);
```

Comment:
This does not seem right - is this function used?

(if you keep functions - rename the parameters to comply with the styleguide)

Reply:
Removed the function altogether and replaced by inlined implementation which makes the code clearer at the call site anyway.